 syntax = "proto3";
   //1 密钥协商 //2 密钥校验; // 3 密钥注销
message RequestMsg
{
	int32 cmdType = 1; // 报文类型
	bytes clientId = 2; // 客户端编号
	bytes serverId = 3; // 服务器端编号
	bytes sign = 4;		// 标记
	bytes data = 5;		// 数据
}

message RespondMsg
{
	bool status = 1; // 返回值
	int32 seckeyid = 2; 	// 对称密钥编号 keysn
	bytes clientId = 3; 	// 客户端编号
	bytes serverId = 4; 	// 服务器编号
	bytes data = 5; 	// 服务器端随机数
}